tasks/build-algolia-search.mjs

```diff
@@ -264,7 +264,7 @@ async function tryParseImports(
     if (Object.keys(fragments).length !== 0) {
       // add platform specific fragments to source
       await Promise.all(
-        fragments[platform].map(async (fragment) => {
+        fragments[platform]?.map(async (fragment) => {
```

Comment:
This is unrelated to the issue that was introduced but fixes an existing issue where pages that have fragments for some but not all platforms will only be indexed for the platforms that have fragments and will error out on the other pages.